### ISSUE TITLE
erroneous milestone instead of seg/div

### ISSUE THREAD

**Issue by gcelano** [OpenGreekAndLatin/First1KGreek#1125]
Look at:

https://github.com/OpenGreekAndLatin/First1KGreek/blob/master/data/tlg0015/tlg001/tlg0015.tlg001.opp-grc1.xml#L5623

Here section 6 is encoded as milestone, while before as seg. I guess this is simply an error. I am also wondering whether all cts-urn elments contain type="textpart" (which would be helpful)

**Comment by sonofmun** [comment]
All `<div>` citation elements should be `<div type="textpart">`. If they are not divs, then they probably won't have a `@type` attribute, e.g., for `<l>` or `<p>`. With `<seg>` I am not sure. Be careful, though. If you are using XPATH to find the nodes and you introduce attributes into your search (e.g., `/div[@subtype="textpart"]` instead of just `/div`, it will slow down your code immensely.

**Comment by sonofmun** [comment]
And thanks for finding that error in that file. Those need to be changed to `<seg>` elements.

**Comment by sonofmun** [comment]
There were quite a few problems with this file. I have corrected them and segmented the text correctly (I think). Once it is merged into master (PR #1127), take a look and close this issue if everything looks good.
